test/sourcemaps/samples/typescript/test.js

```diff
@@ -0,0 +1,17 @@
+export function test({ assert, input, preprocessed }) {
+	const content = '<h1>Hello world!</h1>';
+
+	const original = input.locate(content);
+	const transformed = preprocessed.locate_1('<h1>Hello world!</h1>');
+
+	assert.deepEqual(
```

Comment:
Can we add another assertion for stuff inside script here, like "let count = 0" is correctly mapped?
Also the test could contain an interface definition to see if that is just removed without adding lines in the output.

Reply:
Updated typescript test and added `external`. Added TODO comment, need to investigate further...